TeamViewerPS/TeamViewerPS.psd1

```diff
@@ -54,7 +54,7 @@
     # TypesToProcess = @()
 
     # Format files (.ps1xml) to be loaded when importing this module.
-    FormatsToProcess = @('TeamViewerPS.format.ps1xml')
+    FormatsToProcess  = @('TeamViewerPS.format.ps1xml')
```

Comment:
No additional space!

Reply:
was autoformatted by linter to align the = signs